app/src/debugTools/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -13,22 +13,31 @@ import javax.inject.Singleton
 class DebugModule @Inject constructor() {
     private val cryingProbabilityEvents: PublishSubject<Float> = PublishSubject.create()
     private val notificationEvents: PublishSubject<String> = PublishSubject.create()
+    private val soundEvents: PublishSubject<Int> = PublishSubject.create()
 
     fun sendCryingProbabilityEvent(cryingProbability: Float) {
         cryingProbabilityEvents.onNext(cryingProbability)
     }
 
     fun sendNotificationEvent(notificationInformation: String) {
-        val currentDateTimeString = LocalDateTime.now().format(DateTimeFormatter.ofPattern(DATE_TIME_PATTERN))
+        val currentDateTimeString =
+            LocalDateTime.now().format(DateTimeFormatter.ofPattern(DATE_TIME_PATTERN))
         notificationEvents.onNext("$notificationInformation at $currentDateTimeString")
     }
 
+    fun sendSoundEvent(decibels: Int) {
+        soundEvents.onNext(decibels)
+    }
+
     fun debugStateObservable(): Observable<DebugState> {
-        return Observable.combineLatest<String, Float, DebugState>(
+        return Observable.combineLatest<String, Float, Int, DebugState>(
```

Comment:
Note you could use RxKotlin `Observables.combineLatest()` to have sane typing.
https://github.com/ReactiveX/RxKotlin/blob/7a21c3e66e726e7970917e96163581d7b8f267e0/src/main/kotlin/io/reactivex/rxkotlin/Observables.kt#L36

That's an example of usage:
```
Observables.combineLatest(Observable.just(1), Observable.just(""), Observable.just(2f))
            .subscribe { (i: Int, s: String, f: Float) ->
                // ...
            }
```